code/edgebridge/build.gradle

```diff
@@ -207,15 +207,17 @@ task platformFunctionalTestJacocoReport(type: JacocoReport, dependsOn: "createPh
 dependencies {
 
     // Adobe Mobile SDK Core
-    implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
```

Comment:
Would you be wiling to do:
```
// implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
implementation 'com.github.adobe.aepsdk-core-android:core:v2.0.0-alpha.3'
```
That way you can uncomment the line when core 2.0.0 artifacts are published and delete the `-alpha.3` link.